src/theory/datatypes/sygus_simple_sym.h

```diff
@@ -0,0 +1,103 @@
+/*********************                                                        */
+/*! \file sygus_simple_sym.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Simple symmetry breaking for sygus.
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__THEORY__DATATYPES__SIMPLE_SYM_BREAK_H
+#define __CVC4__THEORY__DATATYPES__SIMPLE_SYM_BREAK_H
+
+#include <map>
+#include "theory/quantifiers/sygus/term_database_sygus.h"
+#include "theory/quantifiers/term_util.h"
+
+namespace CVC4 {
+namespace theory {
+namespace datatypes {
+
+/** SygusSimpleSymBreak
+ *
+ * This class implements queries that can be queried statically about sygus
+ * grammars, for example, concerning which constructors need not appear at
+ * argument positions of others. This is used by the sygus extension of the
+ * quantifier-free datatypes procedure for adding symmetry breaking lemmas.
+ * We call this class of techniques "simple static symmetry breaking".
```

Comment:
It'd be nice to say why this is done rather than using the dynamic symmetry breaking